packages/backend/src/videos/videos.controller.ts

```diff
@@ -4,6 +4,7 @@ import { FilterVideoDTO } from './videos.dto';
 import { FilterVideosDtoMaping } from './videos.validation.pipe'
 import { AuthGuard } from '@nestjs/passport';
 import {ApiTags,ApiParam} from '@nestjs/swagger'
+import VideosEntity from 'src/videos/videos.entity';

```

Comment:
Nie importujemy przez src bo to rodzi problemy z testami

Reply:
a dodatkowo u mnie przez src nie działa , maczek ma z tym problem

Reply:
ok, pewnie autoimport